packages/ui/components/styles/Fonts/nunitoFont.style.css

```diff
@@ -0,0 +1,2 @@
+
+  @import url('https://fonts.googleapis.com/css2?family=Nunito:wght@200;300;400;500;600;700;800;900;1000&display=swap');
```

Comment:
Add a new line

Reply:
By the way, why didnt you import the other thing here?
![image](https://github.com/FeedAHelp/FeedAHelpless/assets/3250040/b9bb0746-5946-4548-bc9a-dbc1f56058df)